lib/wlib/memory/Memory.h

```diff
@@ -34,59 +40,136 @@ class MemoryInitDestroy {
     static int m_srefCount;
 };
 
+/**
+ * This code should never be called and/or touched. It is used internally
+ */
 static MemoryInitDestroy g_smemoryInitDestroy;
 
-extern "C" {
-
+/**
+ * NEVER EVER USE THIS OR I WILL COME AND CREATE A NEW VERSION OF YOU

```

Comment:
sassy, I'd put a bunch of underscores `__memory_alloc(uint32_t size)` to make it clear that it's not meant to be an exposed function

Reply:
Lmao yeah will do that as well